deployment/v3/external/README.md

```
@@ -9,6 +9,15 @@
 * Global configmap: In MOSIP `global` configmap in `default` namespace contains Domain related information.
   * Make sure kubeconfig file is already set and k8 cluster is accessible and kubectl is installed.
   * Copy `global_configmap.yaml.sample` to `global_configmap.yaml`.
+  * Retrieve the Kibana Host Entered During Logging Deployment
+    To retrieve the Kibana Host that was entered by the user as part of the logging deployment, run the following command:
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)